src/dbnode/storage/types.go

```diff
@@ -1271,6 +1283,9 @@ type newFSMergeWithMemFn func(
 	dirtySeriesToWrite map[xtime.UnixNano]*idList,
 ) fs.MergeWith
 
+// NewBackgroundProcessFn is a type of function that creates and returns a new BackgroundProcess.
```

Comment:
nit: `NewBackgroundProcessFn is a function that creates and returns a new BackgroundProcess.`

Reply:
ok